hardware/Sterbox.cpp

```diff
@@ -13,7 +13,7 @@
 
 extern CSQLHelper m_sql;
 
-#define STERBOX_POLL_INTERVAL 10
+#define STERBOX_POLL_INTERVAL 8

```

Comment:
I only remove 4 lines and change from 10 to 8. Nothing more. 
Why 8? Because i have movement sensor connected to this device and if 10 seconds then sometimes miss movement signals. With 8 is OK :D